paddlets/utils/utils.py

```diff
@@ -56,13 +57,13 @@ def check_model_fitted(model: Trainable, msg: str = None):
 
     raise_if_not(fitted, msg % {"name": type(model).__name__})
 
-def get_uuid(prefix: str = "", suffix: str = ""):
+def get_uuid(prefix: Optional[str] = "", suffix: Optional[str] = ""):
```

Comment:
Only when default value is None, Optional is need. Refers to:
https://docs.python.org/3/library/typing.html#typing.Optional